src/main/java/aj/Task.java

```diff
@@ -0,0 +1,35 @@
+package aj;
+
+
+/**
+ * Task abstract class for user tasks.
+ */
+public abstract class Task {
+
+    protected boolean isMarked;
+    protected String taskName;
+
+
+    public void toggleComplete() {
+        this.isMarked = !this.isMarked;
+    }
+
+    public boolean isCompleted() {
+        return this.isMarked;
+    }
+
+    public String getTaskName() {
+        return this.taskName;
+    }
+
+
+    @Override
+    public String toString() {
+        return (this.isMarked ? "[X] " : "[ ] ") + this.taskName;
+    }
+
+    Task(String taskName, boolean completed) {
+        this.taskName = taskName;
```

Comment:
Perhaps you could add a JavaDoc comment above the constructor to explain its purpose and parameters?